specs/binding-mqtt.spec/src/main/resources/META-INF/zilla/mqtt.idl

```diff
@@ -254,8 +264,17 @@ scope mqtt
         struct MqttOffsetMetadata
         {
             uint8 version = 1;
-            uint8 length;
-            int16[length] packetIds;
+            int8 length;
+            int16[length] packetIds = null;
+        }
+
+        struct MqttPublishOffsetMetadata
+        {
+            uint8 version = 1;
+            int64 producerId = 0;
+            int16 producerEpoch = 0;
```

Comment:
We can't have (or need) `kafka` concepts in `mqtt` metadata.

Should these be in `internal.idl` of `mqtt-kafka` binding instead?